lib/fog/vsphere/models/compute/server.rb

```diff
@@ -237,6 +239,14 @@ def revert_snapshot(snapshot)
           end
         end
 
+        def cdroms(opts = {})
+          service.cdroms(:instance_uuid => self.id).all(opts)
+        end
```

Comment:
I think, we need to do define our own getter as we need to pass the instance_uuid. Or is there a way set the instance_uuid using the DSL?